### ISSUE TITLE
Create a UI flow for Palomabot Telegram Bot

### ISSUE THREAD

**Issue by taariq** [VolumeFi/paloma#509]

**Comment by taariq** [comment]
@joycehermosilla Please use Unibot and take a look at their control design.

**Comment by joycehermosilla** [comment]
@taariq Yes, I have checked UniBot.  but for confirm, you want to take Bots to a Telegram format?

**Comment by taariq** [comment]
Yarp. We will need a Telegram bot with buttons to reflect Paloma controls. I am thinking right now controls are just links back to website for now to keep it simple.
